src/app/loans/loans-routing.module.ts

```diff
@@ -0,0 +1,69 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { Routes, RouterModule } from '@angular/router';
+
+/** Routing Imports */
+import { Route } from '../core/route/route.service';
+
+/** Translation Imports */
+import { extract } from '../core/i18n/i18n.service';
+
+/** Custom Components */
+import { AddLoanChargeComponent } from './add-loan-charge/add-loan-charge.component';
+
+/** Custom Resolvers */
+import {LoanChargeTemplateResolver} from './common-resolvers/loan-charge-template.resolver';
+
+const routes: Routes = [
+  Route.withShell([
+    {
+      path: ':clientType',
```

Comment:
As  in the screenshot from Community App you can see that we can add Group Loan Applications too.Once they are added we can add loan charge on them which opens up the same view addLoanCharge view which is opened for clients.so I though of making this route as generic route so that it can be accessed for both group loans and clients loans
![screenshot from 2019-01-21 20-54-53](https://user-images.githubusercontent.com/15383669/51483374-edac6200-1dbe-11e9-89bc-4137c8dc9299.png)

In addition to this i would like to suggest a change in client view route too.
![screenshot from 2019-01-21 21-05-43](https://user-images.githubusercontent.com/15383669/51483976-619b3a00-1dc0-11e9-850a-09a91d0e60f4.png)

Here /clients lists all the clients 
and /clients/view/1 lists the client with id=1 Now i feel that /view is unnecessary since it does not open up any new page  /clients/1 can achieve the same results and make the URL shorter.

current URL length
![screenshot from 2019-01-21 21-09-44](https://user-images.githubusercontent.com/15383669/51484226-f43bd900-1dc0-11e9-8a56-949ff8541da5.png)

Reply:
Alright! Good to see you go in-depth to learn about the project while working on the issue. About shortening the URL, yes your suggestion seems good to me. Although it would require refactoring of routes currently configured in all other implemented sections of the app not only here. Let me get back to you on this.

Reply:
Sure

Reply:
@Jov03 Please create an issue suggesting this as an enhancement, mention that it requires an update in all the implemented sections.

Reply:
@abhaychawla sure ill create an issue for the same.